README.md

```diff
@@ -279,8 +279,8 @@ that other nodes can use to connect to it and exchange peer information. Make su
 replace the displayed IP address information (most probably `[::]`) with your externally
 accessible IP to get the actual `enode` URL.
 
-*Note: You could also use a full-fledged `geth` node as a bootnode, but it's the less
-recommended way.*
+*Note: You could also use a full-fledged `geth` node as a bootnode, which is the recommended way, 
+but it is good practice to create a static nodekey with the bootnode utility*
```

Comment:
It's better to use this method for creating the key and ENR for configuration of other nodes.

```shell
$ devp2p key generate node1.key
# file node1.key is created

$ devp2p key to-enr -ip 10.2.3.4 -udp 30303 -tcp 30303 node1.key 
# prints the ENR for use in --bootnode flag
```

Reply:
Ah perfect, will add that